### PR TITLE
XRDDEV-787

### PR DESCRIPTION
**Pull request by raulmartinez-leadin**
- Added custom username and password for the Admin UI login.

### CODE REVIEW

File: sidecar/Dockerfile [hunk 1]
```diff
@@ -9,8 +11,8 @@ RUN apt-get -qq update \
 && adduser --quiet --system --uid 998 --home /var/lib/postgresql --no-create-home --shell /bin/bash --group postgres \
 && adduser --quiet --system --uid 999 --home /var/lib/xroad --no-create-home --shell /bin/bash --group xroad \
 && adduser xroad sudo \
-&& useradd -m xrd -s /usr/sbin/nologin -p '$6$JeOzaeWnLAQSUVuO$GOJ0wUKSVQnOR4I2JgZxdKr.kMO.YGS21SGaAshaYhayv8kSV9WuIFCZHTGAX8WRRTB/2ojuLnJg4kMoyzpcu1' \
-&& echo "xroad-proxy xroad-common/username string xrd" | debconf-set-selections \
+&& useradd -m ${XROAD_ADMIN_USER} -s /usr/sbin/nologin -p $(python3 -c "import crypt; print(crypt.crypt('${XROAD_ADMIN_PASSWORD}', crypt.mksalt(crypt.METHOD_SHA512)))") \
```
Comment (jhyoty) [inline sidecar/Dockerfile]: Maybe unnecessarily complex to use python to create a hashed password here, consider something like:
```suggestion
&& useradd -m -s /usr/sbin/nologin ${XROAD_ADMIN_USER} \
&& echo "${XROAD_ADMIN_USER}:${XROAD_ADMIN_PASSWORD}" | chpasswd \
```

Comment (jhyoty) [reply sidecar/Dockerfile]: btw, the `devscripts` dependency (line 8 in Dockerfile) is unnecessary since the packages are installed from a repository.

Comment (raulmartinez-leadin) [reply sidecar/Dockerfile]: Agree with the changes. Done.

File: README.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ Minimum recommended docker engine configuration to run sidecar security server c
 Run the following script:
 
   ```bash
-  ./setup_security_server_sidecar.sh <name of the sidecar container> <admin UI port> <software token PIN code>
+  ./setup_security_server_sidecar.sh <name of the sidecar container> <admin UI port> <software token PIN code> <admin UI username> <admin UI password>
```
Comment (iluwatar) [inline README.md]: We should describe the new feature as a bullet point in the text below

Comment (raulmartinez-leadin) [reply README.md]: Done. Also changed admin UI -> admin.

**Pull request merged**